backoffice/src/Program.cs

```diff
@@ -74,6 +76,11 @@
 builder.Services.AddScoped<ILetterRepository, LetterRepository>();
 builder.Services.AddScoped<ILetterService, LetterService>();
 
+builder.Services.AddScoped<IGovNotifyClient, GovNotifyClient>();
+
+builder.Services.AddScoped<IS3Client, S3Client>();
+builder.Services.AddScoped<IWreckMaterialService, WreckMaterialService>();

```

Comment:
Another picky thing, re-order these to have the WM service and WMRepo together like the others